Topics/Development_Process/Documentation/Swagger_API_Documentation.md

```diff
@@ -0,0 +1,44 @@
+# API Documentation with SwaggerHub
+
+## Table of Contents
+- [Purpose of API Documentation](#purpose-of-api-documentation)
+- [Keys to Good API Documentation](#keys-to-good-api-documentation)
+- [Introduction of SwaggerHub](#introduction-of-swaggerhub)
+  - [Features](#features)
+- [Other Free Tool](#other-free-tool)
+
+
+## Purpose of API Documentation
+As the name suggests, API documentation describes a handbook of instructions and expectations around working with the project's APIs. 
```

Comment:
maybe give a quick description of what an API is. I'm not too sure a new csc301 student would know what that is

Reply:
Addressed [here](https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/279/commits/374c16346a04a489f48b529dd5db62113158b6bc)

Reply:
I tried to keep it brief so I only explained the parts that were relevant

Reply:
Small typo. Says "andd" but other than that looks good!

Reply:
this was meant for the comment about standardizing. The typo is on line 46

Reply:
> Addressed [here](https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/279/commits/374c16346a04a489f48b529dd5db62113158b6bc)

I think this was a great solution! It goes into just enough detail to provide context but not too much where it's overwhelming or borderline off topic